content/docs/getting-started/README.md

```diff
@@ -312,7 +337,7 @@ metadata:
 spec:
   acme:
     server: https://acme-staging-v02.api.letsencrypt.org/directory
-    email: $EMAIL_ADDRESS # ❗ Replace this with your email address
```

Comment:
The env var here is confusing since it won't be replaced by the shell, unlike all other commands; that's why I propose to replace it with a "placeholder-like"